centralstore/redis_store.go

```diff
@@ -96,6 +96,9 @@ func (r *RedisBasicStore) Start() error {
 
 	// register metrics for each state
 	for _, state := range r.states.states {
+		if state == DecisionDrop || state == DecisionKeep {
```

Comment:
isn't this from https://github.com/honeycombio/refinery/pull/1151?

Reply:
Yes, and this is the metrics registering code which I missed in that PR